pages/index.vue

```diff
@@ -116,7 +116,12 @@ import { logTrackerEvent } from '~/utils/logger'
 
 const iscnModule = namespace('iscn')
 
-@Component
+@Component({
+  fetch({ redirect, query }) {
+    const { q } = query;
+    if (q) redirect({ name: 'search-keyword___en', query });
```

Comment:
can use `localeRoute({ name: 'search-keyword' })` ?

Reply:
typescript doesn't have `localeLocation` in context, how can I hack it..